src/Core/src/VisualDiagnostics/ViewTransform.cs

```diff
@@ -0,0 +1,22 @@
+﻿namespace Microsoft.Maui
+{
+	internal class ViewTransform
```

Comment:
This can be MAtrix4x4 from System.Numerics. Should be available on all platforms since we use it in Maui.Graphics.